Schemas/RP14A.xsd

```diff
@@ -57,6 +57,70 @@
 											</xsd:simpleType>
 										</xsd:element>
 										
+										<xsd:element name="ComponentPayPerWeek" minOccurs="0" maxOccurs="8">
+											<xsd:annotation>
+												<xsd:documentation>Override for base pay when calculating a component rate.</xsd:documentation>
+											</xsd:annotation>
+
+											<xsd:complexType>
+												<xsd:sequence>
+													<xsd:element name="ComponentType" minOccurs="1" maxOccurs="1">
+														<xsd:simpleType>
+															<xsd:restriction base="xsd:string">
+																<xsd:enumeration value="Redundancy Pay"/>
+																<xsd:enumeration value="Arrears of Pay"/>
+																<xsd:enumeration value="Holiday Pay Taken Not Paid"/>
+																<xsd:enumeration value="Holiday Pay Accrued"/>
+																<xsd:enumeration value="Protective Award"/>
+																<xsd:enumeration value="Compensatory Notice Pay"/>
+																<xsd:enumeration value="Refund of Notional Tax"/>
+																<xsd:enumeration value="Basic Award"/>
+																<xsd:enumeration value="Notice Worked Not Paid"/>
+															</xsd:restriction>
+														</xsd:simpleType>
+													</xsd:element>
+													<xsd:element name="ComponentRate" minOccurs="0" maxOccurs="1">
+														<xsd:simpleType>
+															<xsd:union>
+																<xsd:simpleType>
+																	<xsd:restriction base="xsd:decimal">
+																		<xsd:fractionDigits value="2"/>
+																	</xsd:restriction>
+																</xsd:simpleType>
+																<xsd:simpleType>
+																	<xsd:restriction base="xsd:string">
+																		<xsd:whiteSpace value="collapse"/>
+																		<xsd:length value="0"/>
+																	</xsd:restriction>
+																</xsd:simpleType>
+															</xsd:union>
+														</xsd:simpleType>
+													</xsd:element>
+													<xsd:element name="ComponentRateStatus" minOccurs="1" maxOccurs="1" default="Rate provided">
+														<xsd:annotation>
+															<xsd:documentation>If the component rate is available, and if not, why.</xsd:documentation>
+														</xsd:annotation>
+														<xsd:simpleType>
+															<xsd:restriction base="xsd:string">
```

Comment:
We've had a request from the maintenance team of an upstream consumer here to specify a maximum length. I'll add a 50 character limit.

Reply:
The upstream consumer in question is changing their data model, so this is no longer needed.